xstate-chatbot/nodejs/src/machine/gis-details.js

```diff
@@ -1347,12 +1347,63 @@ const gisFlow = {
               let message = dialog.get_message(messages.updateProperty.prompt, context.user.locale);
               dialog.sendMessage(context, message);
             }),
-            target: '#endstate',
+            target: '#anyOtherParsal',
           }
         ],
       },
     },
-  },
+    anyOtherParsal: {
```

Comment:
Spelling not same as other files. You may want to change spelling to "parcel".

Reply:
changed the spelling to 'parcel'